modules/openy_gc_log/src/Entity/LogEntity.php

```diff
@@ -268,6 +269,20 @@ public static function defineChangedField(array &$fields) {
       ->setDescription(t('The time that the entity was last edited.'));
   }
 
+  /**
+   * Define event_metadata field.
+   *
+   * @param array $fields
+   *   Fields.
+   */
+  public static function defineMetadataField(array &$fields) {
+    $fields['event_metadata'] = BaseFieldDefinition::create('string')
+      ->setLabel(t('Event Metadata'))
+      ->setDescription(t('The event metadata, related to the Log entity.'))
+      ->setSetting('max_length', 255)
```

Comment:
let's make it larger in case if we need to put more info to the meta in the future.